llvm/tools/llvm-mc/llvm-mc.cpp

```diff
@@ -559,6 +559,9 @@ int main(int argc, char **argv) {
         std::unique_ptr<MCCodeEmitter>(CE), *STI));
     if (NoExecStack)
       Str->initSections(true, *STI);
+    if (TheTriple.isOSBinFormatMachO() && TheTriple.isOSDarwin())
```

Comment:
Is this needed? It seems that `emitVersionForTarget` will ignore a major version of 0.

Reply:
No. I just put it in front of all emitVersionForTarget for clarity.